website/src/content/docs/pt-br/index.mdx

```diff
@@ -190,7 +188,7 @@ import LinterExample from "@/components/linter/example.md";
   <h2 class="head">Comunidade</h2>
   <div class="community-connect">
     <div class="community-description">
-      Desenvolvido pela nossa incrível comunidade
+      Alimentado por nossos colaboradores de código aberto
```

Comment:
@Exerea I think in this case "Mantido" or "Desenvolvido" is more appropriate here